.gitmodules

```diff
@@ -29,3 +29,6 @@
 	path = SparkTraining
 	url = https://github.com/cerndb/SparkTraining.git
 	branch = master
+[submodule "GPU_and_data"]
+	path = GPU_and_data
+	url = https://github.com/cerndb/NotebooksExamples
```

Comment:
I just realized all the other submodules are specified with .git , not sure that will be required, but better to keep the consistency?